File: CS/CustomAxisLabels/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace CustomAxisLabels {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        #region CustomDrawAxisLabel
        private void chartControl1_CustomDrawAxisLabel(object sender, CustomDrawAxisLabelEventArgs e) {
            AxisBase axis = e.Item.Axis;
            if (axis is AxisY || axis is AxisY3D || axis is RadarAxisY) {
                double axisValue = (double)e.Item.AxisValue;
                if (axisValue < 0)
                    // Customize the axis label's color.
                    e.Item.TextColor = Color.Red;
                else if (axisValue > 0) {
                    // Customize the axis label's text and color.
                    e.Item.Text = "+" + e.Item.Text;
                    e.Item.TextColor = Color.Green;
                }
                else if (axisValue == 0) {
                    // Hide the axis labels.
                    e.Item.Text = "";
                }
            }
        }        
        #endregion
    }
}

File: VB/CustomAxisLabels/Form1.vb
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace CustomAxisLabels

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub chartControl1_CustomDrawAxisLabel(ByVal sender As Object, ByVal e As CustomDrawAxisLabelEventArgs)
            Dim axis As AxisBase = e.Item.Axis
            If TypeOf axis Is AxisY OrElse TypeOf axis Is AxisY3D OrElse TypeOf axis Is RadarAxisY Then
                Dim axisValue As Double = CDbl(e.Item.AxisValue)
                If axisValue < 0 Then
                    e.Item.TextColor = Color.Red
                ElseIf axisValue > 0 Then
                    e.Item.Text = "+" & e.Item.Text
                    e.Item.TextColor = Color.Green
                ElseIf axisValue = 0 Then
                    e.Item.Text = "Zero"
                End If
            End If
        End Sub
    End Class
End Namespace
